code/modules/vehicles/speedbike.dm

```diff
@@ -3,13 +3,11 @@
 	icon = 'icons/obj/bike.dmi'
 	icon_state = "speedbike_blue"
 	layer = LYING_MOB_LAYER
-	var/overlay_state = "cover_blue"
-	var/mutable_appearance/overlay
+	var/cover_iconstate = "cover_blue"
 
 /obj/vehicle/ridden/speedbike/Initialize()
 	. = ..()
-	overlay = mutable_appearance(icon, overlay_state, ABOVE_MOB_LAYER)
-	add_overlay(overlay)
+	add_overlay(image(icon, cover_iconstate, ABOVE_MOB_LAYER))
```

Comment:
isn't it better to cache this? or am I braindead

Reply:
Why would we cache it?